### DISCUSSION TITLE
How to I Host Azure  Ubuntu server

### DISCUSSION THREAD

**Discussion by Mehediiovi** [pocketbase/pocketbase#2098] [Q&A]
How to I Host Azure  Ubuntu server

**Comment by ganigeorgiev** [comment]
The setup shouldn't be much different compared to a regular VPS provider like Hetzner, Vultr, DigitalOcean, etc. and you can follow the minimal 4-step guide from https://pocketbase.io/docs/going-to-production/#minimal-setup.

The main difference would be that when you create your Azure VM you need to attach a persistent disk (SSD or HDD), so that you can store your `pb_data` there.

**Reply by richard457** [reply]
I have a question, where is the DB Path on disk ? so I can do a backup maybe?

**Reply by ganigeorgiev** [reply]
@richard457 
> where is the DB Path on disk ?

`pb_data/data.db`. If you are making a backup while the application is running make sure to also copy the `-wal` and `-shm` files.

You can also check https://pocketbase.io/docs/going-to-production/#backup-and-restore.

**Reply by richard457** [reply]
Thank you very much.

**Reply by Mellbin** [reply]
The only thing not in the docs that's "missing" and it might not happen to you, but i had to bind my "www.xxxx.com" to the ip used in your azure network(10.0.0.4 was mine).
On linux there is a file called something i can't remember now but it's mentioned somewhere here on the forum if you have issues binding your domain.
